File: RecipeAssistant/RecipeAssistant/AssetUtils.cs
using System;
using Android.Content;
using System.IO;
using Android.Util;
using System.Text;
using Org.Json;
using Android.Graphics;
using Java.IO;

namespace RecipeAssistant
{
	public static class AssetUtils
	{
		const string Tag = "RecipeAssistant";

		public static byte[] LoadAsset(Context context, String asset)
		{
			byte[] buffer;
			try 
			{
				using (Stream stream = context.Assets.Open(asset))
				{
					using (MemoryStream ms = new MemoryStream())
					{
						stream.CopyTo(ms);
						return ms.ToArray();
					}
				}
			}
			catch (Exception ex) {
				Log.Error (Tag, "Failed to load asset " + asset + ": " + ex);
				return null;
			}
		}

		public static JSONObject LoadJSONAsset (Context context, string asset)
		{
			string jsonString = Encoding.UTF8.GetString (LoadAsset (context, asset));
			try 
			{
				return new JSONObject(jsonString);
			}
			catch (Exception ex){
				Log.Error (Tag, "Failed to parse JSON asset " + asset + ": " + ex);
				return null;
			}
		}

		public static Bitmap LoadBitmapAsset(Context context, string asset)
		{
			try 
			{
				using (Stream stream = context.Assets.Open(asset))
				{
					if (stream != null)
					{
						return BitmapFactory.DecodeStream(stream);
					}
				}
			}
			catch (Exception ex) {
				Log.Error (Tag, ex.ToString ());
			}
			return null;
		}
	}
}

File: AgendaData/Components/androidwearpreview-1.0.0.1/lib/android/1.0/content/AndroidWearPreview/samples/RecipeAssistant/App/src/main/java/com/example/android/wearable/recipeassistant/AssetUtils.java
package com.example.android.wearable.recipeassistant;


import android.content.Context;
import android.graphics.Bitmap;
import android.graphics.BitmapFactory;
import android.graphics.drawable.Drawable;
import android.util.Log;

import org.json.JSONException;
import org.json.JSONObject;

import java.io.IOException;
import java.io.InputStream;

final class AssetUtils {
    private static final String TAG = "RecipeAssistant";

    public static byte[] loadAsset(Context context, String asset) {
        byte[] buffer = null;
        try {
            InputStream is = context.getAssets().open(asset);
            int size = is.available();
            buffer = new byte[size];
            is.read(buffer);
            is.close();
        } catch (IOException e) {
            Log.e(TAG, "Failed to load asset " + asset + ": " + e);
        }
        return buffer;
    }

    public static JSONObject loadJSONAsset(Context context, String asset) {
        String jsonString = new String(loadAsset(context, asset));
        JSONObject jsonObject = null;
        try {
            jsonObject = new JSONObject(jsonString);
        } catch (JSONException e) {
            Log.e(TAG, "Failed to parse JSON asset " + asset + ": " + e);
        }
        return jsonObject;
    }

    public static Bitmap loadBitmapAsset(Context context, String asset) {
        InputStream is = null;
        Bitmap bitmap = null;
        try {
            is = context.getAssets().open(asset);
        } catch (IOException e) {
            e.printStackTrace();
        }
        if (is != null) {
            bitmap = BitmapFactory.decodeStream(is);
        }
        return bitmap;
    }
}
